protoc-gen-gogo/generator/generator.go

```diff
@@ -784,16 +784,18 @@ func (g *Generator) WrapTypes() {
 		genFileNames[n] = true
 	}
 	for _, f := range g.Request.ProtoFile {
-		// Make sure the registration path matches the import path.
-		// We check this by making sure that the file's name name
-		// starts with the package name.
-		//
-		// Example:
-		// Proto file "google/protobuf/descriptor.proto" should be imported
-		// from OS path ./google/protobuf/descriptor.proto.
-		expectedPrefix := strings.ReplaceAll(*f.Package, ".", "/") + "/"
-		if !strings.HasPrefix(*f.Name, expectedPrefix) {
-			panic(fmt.Errorf("file name %s does not start with expected %s; please make sure your folder structure matches the proto files fully-qualified names", *f.Name, expectedPrefix))
+		if f.Package != nil {
```

Comment:
I added this `if` check on top of #32 to still allow empty package name, so that we don't stray away too much from gogo's original behavior.

Reply:
Can we have a new tag then after this PR?

Reply:
I'd say let's tag after https://github.com/cosmos/gogoproto/pull/37? Around next week. Or we can do 2 consecutive tags too.

Reply:
Alright 👍🏾 I didn't see https://github.com/cosmos/gogoproto/pull/37